.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# CODEOWNERS: <https://help.github.com/articles/about-codeowners/>
+
+# Everything goes through the following "global owners" by default
+# Unless a later match takes precedence, these three will be
+# requested for review when someone opens a PR
+# Note that the last matching pattern takes precedence, so
+# global owners are only requested if there isn't a more specific
+# codeowner specified below. For this reason, the global codeowners
+# are often repeated in package-level definitions
+
+* @evan-forbes
+
+docs/ @liamsi @adlerjohn @MSevey
```

Comment:
@liamsi can you please give @MSevey write access on this repo to resolve this error

<img width="565" alt="Screen Shot 2022-09-27 at 9 46 35 PM" src="https://user-images.githubusercontent.com/3699047/192668749-b274c2e7-d4db-40e2-a61c-db8dea53b48c.png">